lib/oli_web/templates/pow_invitation/mailer/invitation_new_user.html.eex

```diff
@@ -0,0 +1,3 @@
+<%= content_tag(:h3, "Hi,") %>
```

Comment:
maybe consider changing the heading to something like "Collaborator Invitation" so that a user can see a preview of what the email is related to in their inbox preview, similar to the Confirm Password email

Reply:
Done